target/linux/realtek/image/Makefile

```diff
@@ -3,9 +3,14 @@
 include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/image.mk
 
-KERNEL_LOADADDR = 0x80100000
+KERNEL_LOADADDR := 0x80100000
+
+LOADER_LOADADDR := 0x80100000
+LOADER_PLATFORM := realtek
+LOADER_FLASH_START := 0xb4000000
```

Comment:
This seems unnecessary, because we don't build the loader in OKLI mode.

Reply:
Thanks for the review. Removed that part, simplified the integration and pushed a new version.